tests/data_files/Makefile

```diff
@@ -1123,6 +1123,104 @@ cert_md5.crt: cert_md5.csr
 	$(MBEDTLS_CERT_WRITE) request_file=$< serial=6 issuer_crt=$(test_ca_crt) issuer_key=$(test_ca_key_file_rsa) issuer_pwd=$(test_ca_pwd_rsa) not_before=20000101121212 not_after=20300101121212 md=MD5 version=3 output_file=$@
 all_final += cert_md5.crt
 
+# PKCS7 test data
+pkcs7_test_cert_1 = pkcs7-rsa-sha256-1.crt
+pkcs7_test_cert_2 = pkcs7-rsa-sha256-2.crt
+pkcs7_test_file = pkcs7_data.txt
+
+# Generate signing cert
+pkcs7-rsa-sha256-1.crt:
+	$(OPENSSL) req -x509 -subj="/C=NL/O=PKCS7/CN=PKCS7 Cert 1" -sha256 -nodes -days 365  -newkey rsa:2048 -keyout pkcs7-rsa-sha256-1.key -out pkcs7-rsa-sha256-1.crt
+	cat pkcs7-rsa-sha256-1.crt pkcs7-rsa-sha256-1.key > pkcs7-rsa-sha256-1.pem
+all_final += pkcs7-rsa-sha256-1.crt
+
+pkcs7-rsa-sha256-2.crt:
+	$(OPENSSL) req -x509 -subj="/C=NL/O=PKCS7/CN=PKCS7 Cert 2" -sha256 -nodes -days 365  -newkey rsa:2048 -keyout pkcs7-rsa-sha256-2.key -out pkcs7-rsa-sha256-2.crt
+	cat pkcs7-rsa-sha256-2.crt pkcs7-rsa-sha256-2.key > pkcs7-rsa-sha256-2.pem
+all_final += pkcs7-rsa-sha256-2.crt
+
+# Generate data file to be signed, (print binary of HELLO, prevents against new line differences in linux and windows)
+#point directly to echo to use the filesytem command and not a shell implementation, onlt filesystem has `-e` flag
```

Comment:
what does "onlt filesystem has `-e` flag" mean? (also, typo "filesytem" before "command")

Reply:
What it meant to say was `only filesystem has `-e` flag` . I apologize for my many typos in the previous few commits. I figure once we pass the CI, we can go ahead and remove most of these commits. Hopefully, the issue can be solved by the git config setting that changes the end of file on Windows. 

More specifically answering you're question. Apparently, the echo command in Make does not call the same echo that the user uses at the command line. The difference is the one that Make calls does not allow for a `-e` flag while the one in bash does. So, we specifically call the `echo` in the filesystem with its absolute path.

Obviously, these calls to absolute paths should probably not be merged. I was more so hoping to just pass the CI at that point. Again, when we can figure out what helps pass the test cases then we can start removing every commit that doesn't.